back/app/policies/project_policy.rb

```diff
@@ -152,8 +132,7 @@ def shared_permitted_attributes
   end
 
   def permitted_attributes_for_create
```

Comment:
I don't think we need to make the distinction any more between params for create and for update